spec/artsy-eventstream/artsy/publisher_spec.rb

```diff
@@ -29,7 +29,7 @@
       allow(event).to receive(:verb).and_return(nil)
       expect { Artsy::EventService::Publisher.publish(topic: 'test', event: event) }.to raise_error 'Event missing topic or verb.'
     end
-    it 'calls publish on the exchange with proper data' do
+    it 'uses verb as routing key when calling publish on the exchange without passing routing_key' do
```

Comment:
I assumed we could _also_ get the routing key from the message yielded to the Sneakers worker, but if that's impossible than I understand the usefulness of `verb`. Still, it's strange how they overlap... maybe as a general practice this library should include the routing_key in the message data so there's only one hierarchical way of identifying different events?